### ISSUE TITLE
query error "openemr/oe-module-faxsms": "^2.0"

### ISSUE THREAD

**Issue by stephenwaite** [openemr/openemr#3312]
### Describe the bug

hi @adunsulag @sjpadgett trying to set up faxing with `5.0.3 dev` but received following query error
```
Query Error

ERROR: query failed: INSERT INTO `module_faxsms_credentials` (`id`, `auth_user`, `vendor`, `credentials`) VALUES (NULL, ?, ?, ?) ON DUPLICATE KEY UPDATE `auth_user`= ?, `vendor` = ?, `credentials`= ?

Error: Column 'id' cannot be null

/interface/modules/custom_modules/oe-module-faxsms/controllers/AppDispatch.php at 258:sqlStatement
at :saveSetup(Array)
/interface/modules/custom_modules/oe-module-faxsms/controllers/AppDispatch.php at 74:call_user_func(Array,Array)
/interface/modules/custom_modules/oe-module-faxsms/controllers/AppDispatch.php at 53:dispatchActions()
/interface/modules/custom_modules/oe-module-faxsms/controllers/TwilioFaxClient.php at 36:__construct()
/interface/modules/custom_modules/oe-module-faxsms/controllers/AppDispatch.php at 113:__construct()
/interface/modules/custom_modules/oe-module-faxsms/controllers/AppDispatch.php at 119:getServiceInstance()
/interface/modules/custom_modules/oe-module-faxsms/index.php at 15:getApiService()
```

### To Reproduce

follow the install guide :)
https://community.open-emr.org/t/another-new-feature-for-v5-0-2-1-fax-with-sms-notification-module/13041/33
and enter your setup credentials with twilio

### Expected behavior
save twilio credentials and proceed to trying to set up fax module

**Comment by stephenwaite** [comment]
going to do some more testing, must be a simple error in the credentials entered

**Comment by stephenwaite** [comment]
nope, think i've got all of the fields filled with values, pretty sure it's a bug

**Comment by sjpadgett** [comment]
Hi @stephenwaite 
No, the id column is auto inc and set to not null however, even though I send default null in insert shouldn't make far enough to matter at least on current mysql version i'm running. Still it's improper usage. Try replace NULL in sql at L-257 in interface/modules/custom_modules/oe-module-faxsms/controllers/AppDispatch.php

I changed v2 to store encrypted credentials in table.

**Comment by sjpadgett** [comment]
Merged the fix to v2.0.0. Thanks @stephenwaite
